indico/htdocs/css/Default.css

```diff
@@ -6783,3 +6783,7 @@ ul.speaker_list > li .contributions li {
     width: 400px;
     height: 200px;
 }
+
+input[name="price"].ng-invalid-required {
```

Comment:
I wouldn't add new css rules to `Default.css`. I think this is the place to put it: `htdocs/sass/modules/_registrationform.scss`.